accounts/admin.py

```diff
@@ -23,3 +24,10 @@ class ProfileAdmin(admin.ModelAdmin):
     fields = ("account", "fullname", "phone", "address", "image")
     search_fields = ("fullname", "address")
     readonly_fields = ("account",)
+
+@admin.register(Connection)
+class ConnectionAdmin(admin.ModelAdmin):
+    list_display = ("account", "name", "relation", "age", "address", "biograghy", "created_at", "updated_at")
```

Comment:
- these fields are too much to display, display `name`, `account`, `age`, and `relation` respectively.
- custom `account` column to view the the name if it wasn't

Reply:
done